src/extension/features/toolkit-reports/pages/balance-over-time/BalanceOverTime.jsx

```diff
@@ -14,15 +16,24 @@ import {
   NUM_DATAPOINTS_LIMIT,
 } from './utils';
 
-import { getEntityManager } from '../../../../utils/ynab';
 export const BalanceOverTimeComponent = ({ allReportableTransactions, filters }) => {
+  const LOCALSTORAGE_PREFIX = 'tk-balance-over-time';
   const GROUPED_LABEL = 'Selected Accounts';
   const TRENDLINE_PREFIX = 'Trendline for ';
 
   // Options to group accounts, use a step graph and/or generate trendlines
-  const [shouldGroupAccounts, setShouldGroupAccounts] = useState(false);
-  const [useStepGraph, setUseStepGraph] = useState(true);
-  const [useTrendLine, setUseTrendLine] = useState(false);
+  const [shouldGroupAccounts, setShouldGroupAccounts] = useLocalStorage(
+    `${LOCALSTORAGE_PREFIX}.shouldGroupAccounts`,
```

Comment:
i'd rather just have `useLocalStorage` use the toolkit prefix that we use in our other local storage utilities if you wouldn't mind changing that up?

Reply:
Yeup I can do it that. 👍 
I'll add the prefix in the hook and so it'll automatically be added and whoever is using this hook won't have to worry about adding it.